### PR TITLE
⭐️ Policy data source

### PR DESCRIPTION
**Pull request by mati007thm**
We created two policy data sources:
1. To list only active policies in a space

```
data "mondoo_active_policy" "policy" {
  space_id = "your-space-1234567"
}
```

2. To list active/inactive policies and querypacks within a space

```
data "mondoo_policy" "policy" {
  space_id      = "your-space-1234567"
  catalog_type  = "ALL" # availabe options: "ALL", "POLICY", "QUERYPACK"
  assigned_only = true
}
```

### CODE REVIEW

File: docs/data-sources/active_policy.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+---
+
+# mondoo_active_policy (Data Source)
+
+Data source that reports all active policies in a space
+
+## Example Usage
+
+```terraform
+data "mondoo_active_policy" "policy" {
```
[the remark this answers is not in the document]
Comment (mati007thm) [reply docs/data-sources/active_policy.md:16]: You are right we would only need this one option, but during development we first designed it with `mondoo_active_policy` and only later discovered the gal-query for `mondoo_policy`. Therefore we left both of them in the PR for the time being.

Should we delete `mondoo_active_policy`?

[the remark this answers is not in the document]
Comment (chris-rock) [reply docs/data-sources/active_policy.md:16]: Yes, lets remove it since it will be confusing for users.

[the remark this answers is not in the document]
Comment (Pauti) [reply docs/data-sources/active_policy.md]: I removed the `mondoo_active_policy` data source and resolved merge conflict ✅

**Review by chris-rock — APPROVED** [review]
Thank you @mati007thm @Pauti

**Pull request merged**